spec.yaml

```diff
@@ -82,6 +82,69 @@ security:
   - AuthorizationHeaderToken: []
 
 paths:
+  /v4/appcatalogs/:
+    get:
+      operationId: getAppCatalogs
+      tags:
+        - app katalog
```

Comment:
Do we have a reason that sometimes we say it `Katalog`? (German?)

Reply:
It's to distinguish our offering on the market, so `App Katalog` is kind of the brand name of this feature, which makes it recognisable as something from Giant Swarm.